packages/cryptoplease_ui/lib/src/timeline_widget.dart

```diff
@@ -0,0 +1,183 @@
+import 'package:cryptoplease_ui/cryptoplease_ui.dart';
+import 'package:flutter/material.dart';
+
+class CpTimelineItem {
+  CpTimelineItem({
+    required this.title,
+    this.subtitle,
+    this.trailing,
+  });
+
+  final String title;
+  final String? subtitle;
+  final String? trailing;
+}
+
+enum CpTimelineStatus { inProgress, success, failure }
+
+class CpTimeline extends StatelessWidget {
+  const CpTimeline({
+    super.key,
+    required this.items,
+    required this.status,
```

Comment:
What about `lastItem` parameter? Right now it's not possible to set which item is "active".